rust/agama-dbus-server/src/questions.rs

```diff
@@ -227,10 +239,35 @@ impl Questions {
         Ok(())
     }
 
-    /// sets questions to be answered by default answer instead of asking user
-    async fn use_default_answer(&mut self) -> Result<(), Error> {
-        log::info!("Answer questions with default option");
-        self.answer_strategies.push(Box::new(DefaultAnswers {}));
+    /// property that defines if questions is interactive or automatically answered with
+    /// default answer
+    #[dbus_interface(property)]
+    fn interactive(&self) -> bool {
+        let last = self.answer_strategies.last();
+        if let Some(real_strategy) = last {
+            real_strategy.id() == DefaultAnswers::id()
+        } else {
+            false
+        }
+    }
+
+    #[dbus_interface(property)]
+    fn set_interactive(&mut self, value: bool) {
+        log::info!("set interactive to {}", value);
+        if value {
+            if !self.interactive() {
+                self.answer_strategies.pop();
```

Comment:
This means that no strategies mean "no non-interactive" strategies, that is, the default strategy is interaction? Should be explicitly stated at Questions::answer_strategies declaration

Reply:
yes, no strategies means that user need to answer it itself. Will document it.